GitCommands/Git/GitModule.cs

```diff
@@ -2725,9 +2725,19 @@ public bool IsDirtyDir()
                 : null;
         }
 
+        /// <summary>
+        /// Get the file contents in HEAD for the file
+        /// </summary>
+        /// <param name="file">The Git status item</param>
+        /// <returns>async Task with contents</returns>
         public async Task<string?> GetFileContentsAsync(GitItemStatus file)
```

Comment:
This should be:
```suggestion
        public async Task<string?> GetFileContentAsync(GitItemStatus file)
```

Reply:
I suggest to keep the s
https://english.stackexchange.com/questions/56831/file-content-vs-file-contents